src/lib/adapters/Nextcloud.js

```diff
@@ -1,6 +1,6 @@
-/* @jsx el */
```

Comment:
This is probably unintended, no?

Reply:
I will replace that, I did not intend to change nextcloud.js in anyway.